client/src/Redux/reducers/gameReducers.js

```diff
@@ -0,0 +1,29 @@
+import { ADD_PLAYER, REMOVE_PLAYER, UPDATE_PLAYER_LIST } from "../../utilities/constants"
+
+
+
+
+const intialState = {
+    players: []
+}
+
+export default function gameReducer(state = intialState, action) {
+
+    console.log("In game reducer", action)
+
+    switch (action.type) {
+        case UPDATE_PLAYER_LIST:
+            return {
+                ...state,
+                players: action.payload
```

Comment:
include players as a subfield in payload. It would be easy to understand that payload contains players.